### PR TITLE
LPD-32099 Test Automation - Guest user can view the document even when guest user doesn't have view permission to the file's parent folder

### PR DESCRIPTION
**Pull request by jkappler**
https://liferay.atlassian.net/browse/LPD-28463

### CODE REVIEW

File: modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/internal/security/permission/resource/test/DLFileEntryModelResourcePermissionTest.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
+
+		_regularUser = UserTestUtil.addUser();
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/internal/security/permission/resource/test/DLFileEntryModelResourcePermissionTest.java:56]: why this name?

Comment (jkappler) [reply modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/internal/security/permission/resource/test/DLFileEntryModelResourcePermissionTest.java:56]: I can change, but it's just a regular user :P

Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/internal/security/permission/resource/test/DLFileEntryModelResourcePermissionTest.java]: regular user is still a _user

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**